devices/stm32f301.yaml

```diff
@@ -167,6 +167,7 @@ _include:
  - ../peripherals/gpio/gpio_v2.yaml
  - common_patches/f3_gpio_ospeedr.yaml
  - common_patches/tim/tim_o24ce.yaml
+ - common_patches/tim/tim13_missing_opm.yaml
```

Comment:
RM0366 Rev 4 doesn't even include a TIM13/14(!) Removing the whole peripheral isn't needed for this PR, but no need to change it